1_fetch/src/write_data.R

```diff
@@ -0,0 +1,7 @@
+library(readr)
+
+
+write_to_csv <- function(data, outfile){
+  write_csv(data, file = outfile)
+  return(outfile)
+}
```

Comment:
I wondered whether this function is so small that it doesn't warrant a separate function call. However, I think the alternative would be to code the `write_csv` function within each target since `targets` is looking for the outfile path:

    tar_target(p1_daily_data_csv,  
    {write_to_csv <- function(data,outfile){  
    readr::write_csv(data,outfile)  
    return(outfile)}   
    write_to_csv(p1_daily_data,"1_fetch/out/daily_do_data.csv")  
    },  
    format = "file")

Since this would have to be repeated for p1_inst_data_csv, I think the separate `write_to_csv` file is more readable and can be repeated for the daily and inst datasets.

Reply:
Yeah that was exactly the thought process I went through too. Glad to know I wasn't too off-base. I think it's just enough of a difference from the `readr` function that it would kind of clog up the pipeline file.